ecl/hql/hqlexpr.cpp

```diff
@@ -5188,6 +5188,24 @@ CHqlExpression *CHqlExpressionWithType::makeExpression(node_operator _op, ITypeI
     return (CHqlExpression *)e->closeExpr();
 }
 
+CHqlExpression* CHqlExpressionWithType::makeExpression(node_operator op, ITypeInfo *type, const std::initializer_list<IHqlExpression *> &operands)
+{
+    HqlExprArray args;
+    args.ensure(operands.size());
```

Comment:
Does the fact that you skip nulls make this ensure() less desirable?

Reply:
Well spotted.  I debated the same point, and thought it is was probably ok.  I could count them first..